docs/UsersGuide.asciidoc

```diff
@@ -554,6 +554,19 @@ Steps 4. to 6. can be repeated for further +assert_screen+s/needles without rest
 test.
 
 
+=== Job group editor gh#2111 ===
+
+Scenarios are defined as part of a job group. The `Edit job group` button exposes the editor.
+
+By default the scenarios are listed in a table per medium, with priority and machine(s).
+Values are changed on the fly. Scenarios are removed by removing all of their machines.
+The `Edit YAML` button can be used to reveal the YAML editor and migrate a group. After
+saving for the first time, the group will only be managed in YAML.
+
+YAML-only groups need to be updated through the YAML editor.
+
+Settings can be specified as a key/value pair in the YAML for each scenario.
```

Comment:
I would strip the "in the YAML" part and better: "For the YAML format settings can be specified as a key/value pair for each scenario."